packages/nimble-components/src/wafer-map/specs/features/change-highlight.md

```diff
@@ -0,0 +1,81 @@
+# Nimble Wafer Map Die Highlighting
+
+## Problem Statement
+
+The wafer map receives an array of strings as the values that will be highlighted.
+
+```
+@observable public highlightedValues: string[] = [];
+```
+
+In the following case the highlighted values are 2 and 3.
+
+![Highlighted Values](resources/highlighted-values.png)
+
+The current approach is limited. We can either highlight all the dies having 2s and 3s or neither of them. The customer needs exceeds this approach. The customer may see an anomaly on the dies having DieX: 15 and Soft Bin: 2 and wishes to highlight all these values, but the current code is not capable of this.
+
+## Links To Relevant Work Items and Reference Material
+
+N\A
+
+## Implementation / Design
```

Comment:
Reminder that, in addition to the code changes and corresponding tests, it would be good to update Storybook to document the new field and include an example. One way to do this would be to add a new option to the `dies` control that includes a set of values with some of them highlighted.

![image](https://github.com/ni/nimble/assets/10500124/074da598-a648-4fda-834f-7726351f1531)

Reply:
Good point, I left a comment inside the spec specifying that we need to changes the tests and also the storybook.